recipe/meta.yaml

```diff
@@ -53,8 +53,12 @@ outputs:
         - make
         - pkg-config
       host:
+      run:
+        - cudatoolkit {{ cuda_compiler_version }}  # [cuda_compiler_version not in ("None", "11.2")]
+        - cudatoolkit >=11.2,<12                   # [cuda_compiler_version == "11.2"]
       run_constrained:
         - ucx-proc * {{ ucx_proc_type }}
+        
```

Comment:
nit: Let's drop this blank line since it is part of the `ucx` package above. Blank lines are used to separate packages in `outputs`

```suggestion
```